Framework/Core/src/ExternalFairMQDeviceProxy.cxx

```diff
@@ -366,6 +365,8 @@ InjectorFunction incrementalConverter(OutputSpec const& spec, uint64_t startTime
       dh.dataOrigin = matcher.origin;
       dh.dataDescription = matcher.description;
       dh.subSpecification = matcher.subSpec;
+      /// No one sets the orbit for us, so we invent one.
+      dh.firstTForbit = *timesliceId;
```

Comment:
Don't forget that any new generation of data must set the same firstTForbit as the origin data.

Reply:
AFAIK the IncrementalConverter is just a testing tool and not really used for the proxy.
If all data comes from input proxies, then the provided orbit is used.

But yes, I think this can be a problem if we have enumerating/timed publishers in the workflow.

Reply:
Yes, it's not the case here, but we definitely must make sure that the new generation uses the same firstTForbit of the inputs.